lib/rewrite/rewrite-set.c

```diff
@@ -43,7 +51,7 @@ log_rewrite_set_process(LogRewrite *s, LogMessage **pmsg, const LogPathOptions *
   GString *result;
 
   result = g_string_sized_new(64);
-  log_template_format(self->value_template, *pmsg, NULL, LTZ_LOCAL, 0, NULL, result);
+  log_template_format(self->value_template, *pmsg, &self->value_template_options, LTZ_SEND, 0, NULL, result);
 
```

Comment:
I was thinking about the LTZ_SEND value. First I disagreed, but came to the conclusion that it's a good call. Set creates values that are being sent, and if no local setting is in place, the default should be send-time-zone().

So good call.